client/profile-wizard/steps/store-details/index.js

```diff
@@ -378,6 +380,7 @@ export default compose(
 			getProfileItems,
 			isOnboardingRequesting,
 		} = select( ONBOARDING_STORE_NAME );
+		const { isResolving } = select( OPTIONS_STORE_NAME );
```

Comment:
Not sure if this is an issue here, but I _have_ seen some weirdness around using `isResolving` due to its async nature. Sam has a P2 post on in [here](https://teamisotope.wordpress.com/2020/11/11/data-resolution-status-with-wp-data/) as well, where he mentions using `hasFinishedResolution` as an alternative.

Reply:
Thanks for bringing this up, Joel!  Updated the selectors where possible.  This one in store details I could not update without a significant refactor because it doesn't actually request the option so it will remain permanently resolving.

We probably need to extract some of the `UsageModal` logic, but I'll leave that for another PR.